.github/workflows/pre-main.yaml

```diff
@@ -190,6 +199,7 @@ jobs:
           path: |
             ${{ env.TNF_OUTPUT_DIR }}/*.xml
             ${{ env.TNF_OUTPUT_DIR }}/claim.json
+            script/claimjson.js
```

Comment:
Does it make more sense to copy the html and js file to the output dir? Also, why not do this in run-cnf-suites.sh so that it happens for any type of test run (github, dev, container)?

Reply:
It's just because you made it too pretty :)